ui/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/ManagementActor.scala

```diff
@@ -115,6 +115,12 @@ class ManagementActor(managers: ProcessingTypeDataProvider[ProcessManager],
       }
     case DeploymentStatus =>
       reply(Future.successful(DeploymentStatusResponse(beingDeployed)))
+
+    case (action: CustomActionRequest, id: ProcessId, user: LoggedUser) =>
```

Comment:
I think we need some validation whether process is in correct state. Of course, it could well be checked also in ProcessManager, but I think here it also wouldn't be bad?

Reply:
In fact, I was planning to add some validation, but I wasn't happy with placing it neither in ProcessManager nor in ManagementActor, 
but, ur right, it should be somewhere, so ManagementActor is probably a better suited for this.